examples/02-inputs/04-forms/src/app.gleam

```diff
@@ -40,49 +41,52 @@ type LoginData {
   LoginData(username: String, password: String)
 }
 
-fn decode_login_data(values: List(#(String, String))) -> Result(LoginData, Form) {
-  // we use the `formal` package to validate and turn the raw list of tuples
-  // we receive from the browser into a Gleam custom type.
-  form.decoding({
-    use username <- form.parameter
-    use password <- form.parameter
-    LoginData(username:, password:)
+fn login_form() -> Form(LoginData) {
```

Comment:
Could we rename this `new_login_form` or `init_login_form` or something similar?